### DISCUSSION TITLE
I need help changing sprite of Drop Dash Dust

### DISCUSSION THREAD

**Discussion by RepeatedKibbles** [Techokami/SonicWorldsNext#109] [Q&A] [answered]
![DropDashDust](https://github.com/Techokami/SonicWorldsNext/assets/101306285/fad410a7-1460-4838-9a12-53ab53415d30)

**Comment by DimensionWarped** [comment]
That's not a complete statement of whatever your specific issue is.

**Reply by RepeatedKibbles** [reply]
I couldn't find tscn of Drop Dash Dust (sorry if I sounded mean).

**Comment by DimensionWarped** [comment]
There is no tscn for it... it's just a png file. You need to make sure to keep with the same dimensions and frame counts/arrangements as the original file if you want it to be easy. Otherwise you are going to have to track it down to either a tscn file or a script (possibly both) and make substantial edits to make it work.

Easiest way to find any mention of a file in another file btw is to learn grep.

EDIT: Went ahead and looked up all of its occurrences for you -- you can find it referenced in a pretty clear and obvious way in GenericParticle.tscn and in RotatingParticle.tscn -- it's also referenced in Player.tscn, but its use there is a lot less obvious to me... seems to just be registered as a dependency, but I can't figure out why really.

**Comment by DimensionWarped** [comment]
After digging a little more (and with a little help from @Renhoex), I can confirm that you just need to make sure the animation for DropDash in GenericParticle.tscn is right and it should work fine in your game.

![image](https://github.com/Techokami/SonicWorldsNext/assets/7477617/f24cd8e6-9737-440f-b14a-4c5e5ea67071)

**Reply by RepeatedKibbles** [reply]
![vlcsnap-2023-11-22-18h42m19s308](https://github.com/Techokami/SonicWorldsNext/assets/101306285/9a1af398-11a3-4a43-bf2e-59f5234c39fb)
For some reason, drop dash dust looks like under a little bit (sorry if I sounded mean).

**Reply by DimensionWarped** [reply, marked as answer]
You either need to move the sprite in the sprite sheet or else realign it in the animator or something along those lines.
